beetmoverscript/utils.py

```diff
@@ -66,3 +66,13 @@ def generate_candidates_manifest(context):
     log.info(pprint.pformat(manifest))
 
     return manifest
+
+
+def amend_props(context, platform_mapping):
+    """Function to alter the `stage_platform` field from balrog_props to their
+    corresponding correct values for certain platforms"""
+
+    _platform = context.properties["stage_platform"]
+    # for some products/platforms this mapping is not needed, hence the default
+    context.properties["stage_platform"] = platform_mapping.get(_platform,
```

Comment:
maybe we shouldn't overwrite `stage_platform` and instead save the new platform prop as just `platform`? That way it's clear that we have two variants of the platform name? I see in `infer_template_args` you already rename it to just `platform`